impedance/preprocessing.py

```diff
@@ -89,6 +89,7 @@ def readGamry(filename):
 
     f, Z = [], []
     for line in raw_data:
+        line = line.replace(",", ".")
```

Comment:
I like how simple this is relative to something like setting the users locale (https://github.com/ECSHackWeek/impedance.py/issues/214). Relies on the fact that you should never have a comma except when it is used as a decimal separator which I think it a fairly good assumption

Do you think this would work for you @Sergio-PY? Anything we're missing here?

Reply:
I just thought of one thing. Is comma used as a thousand separator in some languages?

Reply:
It is in written language (e.g. "1 kilogram is 1,000 grams") but I've never seen that formatting used in data files. So I'm inclined to proceed with this and work out a more sophisticated solution to handle the thousand separator case if it comes up.

Reply:
Many apologies for the delay in the reply. The suggested change works perfectly fine although it was for the biologic read function.